downstream/modules/platform/proc-controller-create-job-template.adoc

```diff
@@ -135,7 +135,7 @@ For more information and examples see link:https://docs.ansible.com/ansible/late
 * *Privilege Escalation*: If checked, you enable this playbook to run as an administrator.
 This is the equivalent of passing the `--become` option to the `ansible-playbook` command.
 * *Provisioning Callbacks*: If checked, you enable a host to call back to {ControllerName} through the REST API and invoke the launch of a job from this job template.
-For more information, see link:https://docs.google.com/document/d/1lzlLLP-4eIRPSs4LsHeRoerKHcVWALvGE5uyIXd_rz4/edit[Provisioning Callbacks].
+For more information, see link:https://www.redhat.com/en/blog/satellite-and-ansible-tower-integration-part-2-provisioning-callbacks[Provisioning Callbacks].
```

Comment:
Access denied to the google doc. I made a guess as to the next most appropriate link--please double check.

Reply:
Double checked the Google doc and the link is:
https://docs.ansible.com/automation-controller/latest/html/userguide/job_templates.html#ug-provisioning-callbacks

Reply:
I double checked the google doc and the link is:
https://docs.ansible.com/automation-controller/latest/html/userguide/job_templates.html#ug-provisioning-callbacks